atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -45,14 +45,41 @@
 from atomic_reactor import start_time as atomic_reactor_start_time
 from atomic_reactor.plugin import PreBuildPlugin
 from atomic_reactor.constants import INSPECT_CONFIG
+from atomic_reactor.util import get_docker_architecture, _PREFERRED_LABELS
 import json
 import datetime
 
 
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
 
-    def __init__(self, tasker, workflow, labels, dont_overwrite=("Architecture", ),
+    DEFAULT_ALIASES = {
+        'Name': 'name',
+        'Version': 'version',
+        'Release': 'release',
+        'Architecture': 'architecture',
+        'BuildDate': 'build-date',
```

Comment:
I don't think we did, although I don't think there is a way to know that for sure.

Anyway, it was added in https://github.com/projectatomic/ContainerApplicationGenericLabels/commit/b291b5f3f4636904ea2f01a2c5759898351c8545.